test/ts-runner.spec.ts

```diff
@@ -130,6 +134,38 @@ test(TsRunner.TSX, async () => {
   expect(syncFn(5)).toBe(5)
 })
 
+test(TsRunner.Node, async () => {
+  const { createSyncFn } = await import('synckit')
+
+  let syncFn = createSyncFn<AsyncWorkerFn>(workerJsPath, {
+    tsRunner: TsRunner.Node,
```

Comment:
This is the default runner now, so no need to pass? And it should check whether is it supported first like the following failing case.